### PR TITLE
refactor: compat with stylelint rules

### PR DESCRIPTION
**Pull request by splincode**
## PR Checklist

Please check if your PR fulfills the following requirements:

-   [x] The commit message follows [Conventional Commits](https://www.conventionalcommits.org/en/)
-   [ ] Tests for the changes have been added (for bug fixes / features)
-   [ ] Docs have been added / updated (for bug fixes / features)

## PR Type

What kind of change does this PR introduce?

-   [ ] Bugfix
-   [ ] Feature
-   [x] Refactoring
-   [ ] Code style update
-   [ ] Build or CI related changes
-   [ ] Documentation content changes

## What is the new behavior?

compatibility with

```
"meowtec/no-px": [
            true,
            {
                "ignore": [
                    "-5px",
                    "-4px",
                    "-3px",
                    "-2px",
                    "-1px",
                    "0px",
                    "1px",
                    "2px",
                    "3px",
                    "4px",
                    "5px"
                ]
            }
        ]
```

## Does this PR introduce a breaking change?

-   [ ] Yes
-   [x] No

### CODE REVIEW

**Comment by splincode** [comment]
@nsbarsukov @vladimirpotekhin please, review me)

File: projects/core/components/link/link.style.less [hunk 1]
```diff
@@ -124,6 +124,8 @@
         padding-bottom: 0.15em;
         text-decoration: underline dashed;
         text-underline-offset: 0.2em;
+        // TODO: research if it can be used in rem units
+        /* stylelint-disable-next-line */
         text-decoration-thickness: 0.7px;
```
Comment (nsbarsukov) [inline projects/core/components/link/link.style.less:129]: it cannot be used in rem units (

Can stylelint - plugin exclude float values between [-5, 5] (not only integers) ?

**Review by nsbarsukov — APPROVED** [review]

File: projects/core/components/link/link.style.less (hunk 1 above)
Comment (splincode) [reply projects/core/components/link/link.style.less:129]: not supported (( only fixed values

**Comment by splincode** [comment]
@waterplea please, review me))

**Review by vladimirpotekhin — APPROVED** [review]

**Pull request merged**